src/global/config.ts

```diff
@@ -1,5 +1,6 @@
 export class Config {
     public iconPath = '';
+    public featureSwitches = {};
```

Comment:
I think this has to be typed as `any`, or the compiler will complain about properties not existing on type `Object`…

Reply:
Oh, yeah! :smile: 
Good catch!

Reply:
Confirmed in testing, so I ⚡️'d it 😄

Reply:
I approve! :heavy_check_mark: :smile: